fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/fragments/ElectionSetupFragment.java

```diff
@@ -94,39 +91,39 @@ public View onCreateView(
       @Nullable ViewGroup container,
       @Nullable Bundle savedInstanceState) {
 
-    mSetupElectionFragBinding = ElectionSetupFragmentBinding.inflate(inflater, container, false);
+    ElectionSetupFragmentBinding setupElectionFragBinding =
```

Comment:
In the rest of the app (where I renamed it too)I named the Binding `binding`. I feel like it is self-explanatory in the context, and it is enough.

What do you think ?

Reply:
Much better I'll change it